### ISSUE TITLE
Using Absolute Positioning with v3 'blocks' does not work correctly

### ISSUE THREAD

**Issue by Gazook89** [naturalcrit/homebrewery#2483]
### Renderer

- [ ] Legacy
- [X] v3

### Browser

Chrome

### Operating System

MacOS

### What happened?

Using v3, the block syntax does not accurately place absolutely positioned elements.  Adding `width:unset` to the inline styling of the block fixes the issue.  Just not sure yet if we need the current `width:100%` on the block, or if it's something that slipped through on a recent update (with themes or something else).

Preview the issue here:   https://homebrewery.naturalcrit.com/share/9GyBDJe9xWIl

### Code

```gfm
<div style="position:absolute;bottom:25px;right:70px;">
html tags syntax
</div>


{{position:absolute,bottom:35px,right:70px
Mustache Syntax
}}
```

**Comment by calculuschild** [comment]
It's probably an accidental inclusion from the Themes update.

**Comment by Gazook89** [comment]
K, it's here:

https://github.com/naturalcrit/homebrewery/blob/ef80c2303481c18bbc33128e0458d1370aa3c06a/themes/V3/Blank/style.less#L222

Can you think of any reason off the bat on why it should be 100%?  Will removing it break other things?  I'll play around with it a little bit but you likely have the better idea on what it is for.  I assume it's for filling a columns width, and that when changing it to absolute positioning it just gets borked.

**Comment by Gazook89** [comment]
So without the width set to 100%, things like notes won't fill the column width.   Maybe it's best to set the width to 100% for each of those types of items  (notes, stat blocks, etc) rather than on the `.block` class.

Or, leave it as-is and just nudge users to utilize `spans` rather than divs, or manually unset width as I suggested in the recent reddit post.

**Comment by calculuschild** [comment]
I suspect you are correct. I likely was trying to simplify the CSS by making all Blocks fill the column, but didn't account for absolute-positioned elements. You might compare to the v3.2.0 branch from before the Themes update to see how any `width: 100%` rules used to be set up.

**Comment by ericscheid** [comment]
Generally in CSS, div blocks _are_ blocks, and thus do fill 100% of the available width. And it's the absolutely positioned blocks that need specifying of width (along with positioning). They are blocks, unmoored from their container — they need positioning and sizing.

Reversing the onus will be rather weird.

**Comment by calculuschild** [comment]
> Reversing the onus will be rather weird.

Yeah it's weird, but to some extent we need to reproduce the previous behavior so old brews with custom blocks work as they did before.

**Comment by Gazook89** [comment]
[I'm not 100% sure I'm making a correct historical comparison here](https://github.com/naturalcrit/homebrewery/compare/3.1.1...master#diff-fee87a312d0e6da8aaa0cb04e7f4352d402a594cffc923b57214471535838f14) but I don't see evidence that `width` for the `.page .block` selector changed at all?  It just got moved into the Blank theme, but it's value hasn't changed for a long time.  Maybe there is something else going on or I just haven't chosen an apt comparison or gone back far enough.

**Comment by calculuschild** [comment]
Huh.... I'm looking at v3.2.0 on the staging site now and we have the same behavior. I think you are right that this isn't actually a new change.

> Generally in CSS, div blocks are blocks, and thus do fill 100% of the available width. 

If this is the case, our `.block` divs are using `display:inline-block` so they aren't acting as standard blocks. I'm sure I did that for a reason when designing the V3 CSS (I think it had to do with column breaking/wrapping) but it meant having to manually assign `width:100%` to get Notes/etc. to fill the column.

Chrome has developed quite a bit since then, and especially with V108 coming out at the end of Nov 2022, a lot of those column-related issues should be fixed. *If that was the reason I made it inline-block*, maybe the easiest fix is to just change `.block` back to `display:block`. Would require some extensive testing to make sure we aren't inadvertently breaking something....

**Comment by Gazook89** [comment]
So I tried the following on a few of my own brews, on some recent shared brews on DOMT server, and with a handful of Kaiburr's templates, and didn't see any ill-effects:

```
.page .block {
  display: block;
  width: unset;
}
```

Every block I saw had it's own `width` property set with greater specificity than `.page .block`.  And I didn't see any issues with turning the display to `block` rather than `inline-block`.  

Currently, the original issue still stands, so the above change to CSS fixes that and I didn't see any negative impact in ~12 brews with moderate complexity.  Of course, that's a tiny sample.  And sometimes users do weird things in their brews.

I'm just combing through my old issues-- this is not urgent.

**Comment by dbolack-ab** [comment]
Could this be better served by creating an "absolute" class to add in this situation?

**Comment by 5e-Cleric** [comment]
Whatever the result of this is, it won't be to change the .block styling, since we don't want to alter all of our users' brews, If it was up to me i'd close this issue right now and forget about a behavior that should be understood intuitively, blocks are as wide as they can, if you make it float, give it a width and height.

**Comment by 5e-Cleric** [comment]
Closing this, the users are very much used to this behavior and i see no point in changing it now. Works great for tables and default blocks.
